Makefile

```diff
@@ -136,8 +145,16 @@ fmt: ; $(info  Running gofmt...) @ ## Run gofmt on all source files
 # To pass proxy for Docker invoke it as 'make image HTTP_POXY=http://192.168.0.1:8080'
 .PHONY: image
 image: | $(BASE) ; $(info Building Docker image...) @ ## Build SR-IOV CNI docker image
+	@echo $(TAG)
 	@docker build -t $(TAG) -f $(DOCKERFILE)  $(CURDIR) $(DOCKERARGS)
 
+push: image
+	docker push $(TAG) \
```

Comment:
We seem to be pushing to both docker hub and ECR here. This will be changed later ?

Reply:
We want to push just one, preferrably docker hub. We can use the credentials that Manasa was using.